build/yamls/base/conf/antrea-agent.conf

```diff
@@ -97,3 +100,6 @@ featureGates:
 # the flow collector.
 # Flow export frequency should be greater than or equal to 1.
 #flowExportFrequency: 12
+
+# Provide the port range used by NodePortLocal for programming IPTABLES rules. This is only used if NodePortLocal is enabled.
+#nplPortRange: 40000=41000
```

Comment:
Is it more common to use "-" as the connector of port range? https://github.com/kubernetes/kubernetes/blob/eca53507be3d9008ff6d4cb776c02733a1ea898b/cmd/kube-apiserver/app/options/options.go#L224
https://docs.docker.com/engine/reference/run/#expose-incoming-ports